mico-core/src/main/java/io/github/ust/mico/core/dto/PrometheusResponseDTO.java

```diff
@@ -19,44 +19,34 @@
 
 package io.github.ust.mico.core.dto;
 
-import java.util.List;
-import java.util.Map;
-
-import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
 import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import io.github.ust.mico.core.util.PrometheusValueDeserializer;
 import lombok.AllArgsConstructor;
 import lombok.Data;
 import lombok.NoArgsConstructor;
 import lombok.experimental.Accessors;
 
+/**
+ * DTO for a response from Prometheus. It contains a status field and the value field for the CPU load / memory usage.
+ */
 @Data
 @NoArgsConstructor
 @AllArgsConstructor
 @Accessors(chain = true)
-@JsonIgnoreProperties(ignoreUnknown = true)
-// TODO: Class comment.
-public class PrometheusResponse {
-    
-    // TODO: Add comments for fields.
-    
-    public static final String PROMETHEUS_SUCCESSFUL_RESPONSE = "success";
+public class PrometheusResponseDTO {
 
+    /**
+     * Status of the response: can be "success" or "error".
+     */
+    @JsonProperty("status")
     private String status;
```

Comment:
Consider an `enum` for the possible values or even use a boolean `success` if there really are only those two values.